fetch.bs

```diff
@@ -7854,21 +7854,19 @@ method steps are:
   <ol>
    <li><p>Set <var>locallyAborted</var> to true.
 
-   <li><p><a>Abort the <code>fetch()</code> call</a> with <var>p</var>, <var>request</var>, <var>responseObject</var>,
-   and <var>requestObject</var>'s <a for=Request>signal</a>'s <a for=AbortSignal>abort reason</a>.
+   <li><p><a for=/>Assert</a>: <var>controller</var> is non-null.
 
-   <li><p>If <var>controller</var> is not null, then <a for="fetch controller">abort</a>
-   <var>controller</var> with <var>requestObject</var>'s <a for=Request>signal</a>'s
-   <a for=AbortSignal>abort reason</a>.
+   <li><p><a for="fetch controller">Abort</a> <var>controller</var> with <var>requestObject</var>'s
+   <a for=Request>signal</a>'s <a for=AbortSignal>abort reason</a>.
```

Comment:
Note to self: this does indeed work. Setting the controller's state will trigger various "abort when" conditions, both inside HTTP network-fetch, HTTP cache-fetch, and blob's part of scheme fetch. All those "abort when" conditions will eventually end up in processResponse with response set to a network error whose the aborted flag set, and that will call "abort the fetch() call".

Reply:
It doesn't actually work for blobs I think because the reading from disk happens in parallel from that. And we don't abort that stream. So I suspect we need to keep all the original invocations. I should probably start a new PR that only does some minor cleanup and perhaps explains how this works.

We should probably also inline the "abort when" thing as a single check at the start of scheme fetch seems sufficient?

Reply:
What we could do here maybe is branch on scheme, to keep the dream of https://github.com/whatwg/dom/issues/1031 mostly alive. Hmm.